src/domain/kaivuilmoitusTaydennys/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { cloneDeep } from 'lodash';
+import { Taydennys } from '../application/taydennys/types';
+import { KaivuilmoitusAlue, KaivuilmoitusData } from '../application/types/application';
+import { mapToKaivuilmoitusArea } from '../kaivuilmoitus/utils';
+
+export function convertTaydennysDataToFormState(taydennys: Taydennys<KaivuilmoitusData>) {
+  const data = cloneDeep(taydennys);
+  data.applicationData.areas = (data.applicationData.areas as KaivuilmoitusAlue[]).map(
```

Comment:
SonarCloud thinks that this could be written as 
```
  data.applicationData.areas = data.applicationData.areas.map(
    mapToKaivuilmoitusArea,
  );
```
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1030&id=City-of-Helsinki_haitaton-ui&open=AZQscfpA3JmFToKTb__x

Reply:
Fixed that.